migrations/20200803012227-create-dish.js

```diff
@@ -24,7 +24,7 @@ module.exports = {
         type: Sequelize.STRING
       },
       tableTalkPoints: {
-        type: Sequelize.STRING
+        type: Sequelize.TEXT
```

Comment:
Do we need to run migration for this when it's merged?

Reply:
I think this is a sequelize-level distinction in terms of how many characters can be in the field